scripts/2.mutation-classifier.py

```diff
@@ -326,19 +326,45 @@ def get_threshold_metrics(y_true, y_pred):
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
+# ### Top coefficients for covariates model
+
 # In[21]:
 
-# Top standardized coefficients
-(coef_df
-    .query("feature_set == 'full'")
-    .head(10)
-)
+coef_df.query("feature_set == 'covariates'").head(10)
 
 
-# ## Investigate the predictions
+# ### Top coefficients for full model
 
 # In[22]:
 
+coef_df.query("feature_set == 'full'").head(10)
```

Comment:
The PCA components are the same; however, in this notebook the `full` model selected 50 components as the best parameter for `n_components` and the `expression` model selected 100 components as the best parameter for `n_components`. 

If we were to try and compare gene coefficients between the two models it would be difficult to determine what effect including covariates had on coefficients because the addition of coefficients isn't the only thing that changed... the number of PCA components also changed.

That said, it would likely still be interesting to see which genes were most impactful to the classifier when including covariates (the `full` model). We may want to present the gene coefficients for the full model in a separate table because doing a side by side comparison with the gene coefficients from the expression only model may make people draw conclusions that aren't there.